test/test_rosbag/test/test_bag.py

```diff
@@ -188,6 +190,29 @@ def test_rosbag_filter(self):
 
         self.assertEquals(len(msgs), 5)
 
+    def test_trivial_rosbag_filter(self):
+        # regression test for issue #736
+
+        inbag_filename  = '/tmp/test_rosbag_filter__1.bag'
+        outbag1_filename = '/tmp/test_rosbag_filter__2.bag'
+        outbag2_filename = '/tmp/test_rosbag_filter__3.bag'
+
+        with rosbag.Bag(inbag_filename, 'w') as b:
+            for i in range(30):
+                msg = ColorRGBA()
+                t = genpy.Time.from_sec(i)
+                b.write('/ints' + str(i), msg, t)
+
+        expression = "True"
+        
+        # filtering multiple times should not affect the filtered rosbag
+        os.system('rosbag filter %s %s "%s"' % (inbag_filename, outbag1_filename, expression))
+        os.system('rosbag filter %s %s "%s"' % (outbag1_filename, outbag2_filename, expression))
```

Comment:
I would suggest to extract the command string into a variable:

    cmd = 'rosbag filter %s %s "True"'

That would avoid the duplication of the string as well as the expression variable.